hummingbot/connector/gateway/clob_perp/data_sources/injective_perpetual/injective_perpetual_constants.py

```diff
@@ -89,3 +91,163 @@ def _parse_network_config_to_denom_meta(config: ConfigParser):
     "testnet": _parse_network_config_to_denom_meta(config=TESTNET_TOKEN_META_CONFIG),
     "devnet": _parse_network_config_to_denom_meta(config=DEVNET_TOKEN_META_CONFIG)
 }
+
+SECOND = 1
+MINUTE = 60
```

Comment:
Please use the globals in hummingbot.connector.constants instead of defining them here